cli/integrity/integrity.go

```diff
@@ -0,0 +1,30 @@
+package integrity
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Signer implements high-level API for operations that require
+// a private key.
+type Signer interface {
+	// Sign generates files with hashes for integrity
+	// checks and signs them.
+	Sign(basePath string, appNameList []string) error
```

Comment:
Up to you, but it is shorter with the same meaning.
```suggestion
	Sign(basePath string, appNames []string) error
```

Reply:
I've changed it